action/store/postgres.py

```diff
@@ -19,6 +22,7 @@ def __init__(self, name: str, database: str, user: str, password: str):
 
     def read(self) -> dict:
         cur = self.conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+        exclude = self.meta["excluded_tables"]
```

Comment:
Redundant variable, just insert this in the execute() argument. There's also already a check on table_name for "migrations", can you merge the two lists?